ft232/libftdi.py

```diff
@@ -23,13 +23,22 @@
 from serial import (FIVEBITS, SIXBITS, SEVENBITS, EIGHTBITS, PARITY_NONE,
                     PARITY_EVEN, PARITY_ODD, STOPBITS_ONE, STOPBITS_TWO)
 
-for lib in ("libftdi.so", "libftdi.so.1", "libftdi.dylib", "libftdi.dylib.1", "libftdi1.dylib"):
+libftdi_filenames = ["libftdi.so", "libftdi.so.1", "libftdi.dylib", "libftdi.dylib.1", "libftdi1.dylib"]
+
+for lib in libftdi_filenames:
     try:
         ftdi = c.CDLL(lib)
+        print("Loaded libftdi: ", lib)
         break
     except OSError:
         continue
 
+try:
+    ftdi
+except:
+    print("Error: Unable to load libftdi!  Please make sure it is available as at least one of the following filenames:\n" + "\n".join(libftdi_filenames))
+    raise Exception("Unable to load libftdi!  Please make sure it is available as at least one of the following filenames:\n" + "\n".join(libftdi_filenames))
```

Comment:
It's poor form to raise a bare Exception seeing it can't be caught by anything. I'd suggest raising a FileNotFoundError instead.